Projects/UOContent/Items/Special/Holiday/Christmas/HolidayTree.cs

```diff
@@ -208,5 +220,15 @@ public override void OnDoubleClick(Mobile from)
                 _tree.OnDoubleClick(from);
             }
         }
+
+        public static void Deserialize(IGenericReader reader, int version)
+        {
+        }
```

Comment:
This should not be needed, instead we use a MigrateFrom.

I can fix up this PR, thank you!